apps/admin/frontend/src/screens/hardware_diagnostics_screen.tsx

```diff
@@ -1,15 +1,63 @@
-import { Font, H2, Icons, P } from '@votingworks/ui';
+import { H2, Icons, P, PrinterRichStatusDisplay } from '@votingworks/ui';
 
 import { format } from '@votingworks/utils';
+import type { PrinterStatus } from '@votingworks/printing';
+import React from 'react';
+import type { DiagnosticsRecord } from '@votingworks/admin-backend';
+import type { BatteryInfo } from '@votingworks/backend';
 import { NavigationScreen } from '../components/navigation_screen';
-import { getPrinterStatus, systemCallApi } from '../api';
+import { getDiagnosticRecords, getPrinterStatus, systemCallApi } from '../api';
 import { Loading } from '../components/loading';
+import { PrintTestPageButton } from '../components/print_diagnostic_button';
+
+function BatteryStatusIcon({ discharging, level }: BatteryInfo): JSX.Element {
+  if (discharging && level < 0.1) {
+    return <Icons.Warning color="warning" />;
+  }
+
+  return <Icons.Done color="success" />;
+}
+
+function PrinterStatusDisplay({
```

Comment:
I think it might be clearer to combine the connection status and the printer status. The default state seems redundant:

> Connected
> Ready

Instead, I was thinking a single line that says:

- When disconnected: No compatible printer detected
- When connected and ready: Ready to print
- Otherwise: <other status description>

Reply:
updated in most recent PR

https://github.com/votingworks/vxsuite/assets/37960853/48cd9e06-3569-42a3-9341-6a9329656461

Reply:
It's interesting, we get a "Stopped - other" blip when exiting sleep mode